src/gui/search/searchjobwidget.cpp

```diff
@@ -128,11 +128,11 @@ SearchJobWidget::SearchJobWidget(SearchHandler *searchHandler, QWidget *parent)
 
     updateFilter();
 
-    m_lineEditSearchResultsFilter = new LineEdit(this);
-    m_lineEditSearchResultsFilter->setPlaceholderText(tr("Filter search results..."));
+    m_lineEditSearchResultsFilter = m_ui->lineEditSearchResultsFilter;
```

Comment:
Why was a new object being create before, but here you can use an existing one?

Reply:
because there was no such object in .ui file. very likely author just didn't know that QtCreator allows to declare custom widget in .ui. see last lines in .ui file for details
I did that just for own convenience, to see and arrange all required controls in designer. I don't like to code UI in general, especially to mix visual and code ways

Reply:
I even plan to drop that variable, I assigned value to it just as "quick solution"